src/apiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getAllMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+     .then(response => response.json())
+}
+
+export const fetchSingleMoviePoster = (id) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+   .then(response => response.json())
+}
```

Comment:
Awesome work refactoring the API calls to be in a separate file. 

It seems the less class based components we have, the faster and more efficient our app will be! Great pattern. Really like this.

Reply:
You could consider holding the base URL in a global variable since it is identical in both functions. You'd just have to add the `id` at the end of one.